cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
This `osxfs` stuff serious engineering and obviously reflects where Docker is putting all their focus: https://beta.docker.com/docs/mac/osxfs/

The docs are a lot more clear about the goals and hint at performance:

> osxfs provides a close-to-native user experience for bind mounting OS X file system trees into Docker containers.

But I do see your point.

* Bind mounts have side effects even if the mac/VM story was flawless.
* Volumes were simply unusable on the last iteration of these tools, and nobody was admitting it
* The osxfs stuff needs to be benchmarked